pcm-player.js

```diff
@@ -2,6 +2,11 @@ function PCMPlayer(option, onendedCallback) {
     this.init(option, onendedCallback);
 }
 
+function getTimestampMs() {
+    var hrTime = process.hrtime();
```

Comment:
we can't rely on this being available in all contexts.  can you sufficiently correct drift using a simpler `Date.now().getTime()`

Reply:
> we can't rely on this being available in all contexts. can you sufficiently correct drift using a simpler `Date.now().getTime()`

Replaced high resolution timer with the common Date API.